browser/_layouts/schema.html

```diff
@@ -42,6 +67,9 @@
         <ul>
             <li><b>Updated:</b> {{ site.time | date: "%B %-d, %Y" }}</li>
         </ul>
+        <label title="Enables table pagination of schema data when checked" style="margin: 20px;">
+          <input type="checkbox" id="toggle-pagination"> Table Pagination
```

Comment:
Please change to "Paginate tables" -- makes clear that this is a user action we are inviting.